docs/products/kafka/howto/enabled-consumer-lag-predictor.md

```diff
@@ -34,24 +34,40 @@ Before you start, ensure you have the following:
    select your project, and choose your Aiven for Apache Kafka® service.
 
 1. On the <ConsoleLabel name="overview"/> page,
-   click <ConsoleLabel name="service settings"/>  from the sidebar.
+   click <ConsoleLabel name="service settings"/> from the sidebar.
 
 1. Go to the **Advanced configuration** section, and click **Configure**.
 
 1. In the **Advanced configuration** window, click <ConsoleIcon name="Add config options"/>.
 
-1. In the add configuration options:
+1. Set `kafka_lag_predictor.enabled` to **Enabled**. This enables the lag
+   predictor to compute predictions for all consumer groups and topics.
 
-   - Set `kafka_lag_predictor.enabled` to **Enabled**. This enables the lag predictor to
-     compute predictions for all consumer groups across all topics.
-   - Set `kafka_lag_predictor.group_filters`  to the desired consumer group pattern. This
-     specifies which consumer groups to consider during lag prediction calculations.
+1. Configure the following options:
 
-     :::note
-     By default, the consumer lag predictor calculates the lag for all
-     consumer groups. To restrict the calculation to specific groups, use
-     the `kafka_lag_predictor.group_filters` option.
-     :::
+   - Set `kafka_lag_predictor.group_filters`: Specify the consumer group pattern to
+     include only the desired consumer groups in the lag prediction. By default, the
+     consumer lag predictor calculates the lag for all consumer groups, but you can
+     restrict this by specifying group patterns.
+
+     **Example**:
+     - `consumer_group_*`: Matches any consumer group that starts with
+       `consumer_group_`, such as `consumer_group_1` or `consumer_group_a`.
+     - `important_group`: Matches the specific consumer group named `important_group`.
+     - `group?-test`: Matches consumer groups like `group1-test` or `groupA-test`, where
+       the `?` represents any single character.
+
+   - Set `kafka_lag_predictor.topics`: Specify which topics to include in the lag
+     prediction. By default, predictions are computed for all topics, but you can
+     restrict this by using topic names or patterns.
+
+     **Example**:
+
+     - `important_topic_*`: Matches any topic that starts with `important_topic_`, such
+       as `important_topic_1`, `important_topic_data`.
+     - `secondary_topic`: Matches the specific topic named `secondary_topic`.
```

Comment:
Exact match for the specific topic named...

Reply:
I've updated the phrasing to: 'Matches exactly the...' to maintain parallelism with the rest of the examples